src/librustc/hir/map/mod.rs

```diff
@@ -160,6 +161,8 @@ impl Forest {
     }
 }
 
+pub(super) type HirMap<'hir> = [Vec<Option<IndexVec<ItemLocalId, Option<Entry<'hir>>>>>; 2];
```

Comment:
`hir::map::Map` *is* the HIR map. Can you rename this type alias to `HirEntryMap` or something?